arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/precompileSubsection/EllipticCurvePrecompileSubsection.java

```diff
@@ -117,6 +117,8 @@ public void resolveAtContextReEntry(Hub hub, CallFrame callFrame) {
           returnData); // TODO @Lorenzo @Olivier : verify it's at the right position
     }
 
+    if (!callSuccess) return;
```

Comment:
**Very important**. Prevents the undesired addition of 2 miscellaneous rows in case of an unsuccessful EC precompile call.